libs/checksum/pom.xml

```diff
@@ -43,16 +43,6 @@
     <build>
         <plugins>
             <plugin>
-                <groupId>org.apache.maven.plugins</groupId>
```

Comment:
For libs these were originally placed here because maven release had issues with releasing from a maven sub-module. Now that these are all the same version should be good to delete.